pyicloud/services/photos.py

```diff
@@ -0,0 +1,286 @@
+import sys
+import json
+import urllib
+
+from datetime import datetime
+from base64 import b64decode
+from bitstring import ConstBitStream
+from pyicloud.exceptions import PyiCloudBinaryFeedParseError
+
+
+class PhotosService(object):
+    """ The 'Photos' iCloud service."""
+
+    def __init__(self, service_root, session, params):
+        self.session = session
+        self.params = dict(params)
+
+        self.prepostfetch = 200
+
+        self._service_root = service_root
+        self._service_endpoint = '%s/ph' % self._service_root
+
+        request = self.session.get(
+            '%s/startup' % self._service_endpoint,
+            params=self.params
+        )
+        response = request.json()
+        self.params.update({
+            'syncToken': response['syncToken'],
```

Comment:
@vjgaero; I'm talking about this `photos` feature.  Without modification, the error users who do not have access to this feature receive is this:

```
Python 2.7.11 (default, Dec  5 2015, 14:44:47)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.1.76)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyicloud import PyiCloudService
>>> svc = PyiCloudService('<username>', '<password>')
>>> svc.photos
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pyicloud/base.py", line 178, in photos
    self.params
  File "pyicloud/services/photos.py", line 29, in __init__
    'syncToken': response['syncToken'],
KeyError: 'syncToken'
```

Earlier (in the above discussion), I dug a little deeper and was able to find that the actual payload returned was the below.  Note that it does not include the key it's expecting (`sync_token`) which is why the error above is being raised, but the underlying problem is actually just that Apple apparently requires payment for this specific feature:

```
{u'errorCode': 402, u'requestUUID': u'4d12d29e-10c8-823a-2cbb-862d9502b72a', u'errorReason': u'Payment Required'}
```